CMakeLists.txt

```diff
@@ -1,5 +1,25 @@
 cmake_minimum_required(VERSION 3.20)  # For using CMAKE_<LANG>_BYTE_ORDER
 
+if(NOT DEFINED USE_VCPKG OR USE_VCPKG)
```

Comment:
`USE_VCPKG` should always be defined if you create the corresponding `option(`

Reply:
> While we're editing the file, we can add that change and remove spdlog ;)

done

> `USE_VCPKG` should always be defined if you create the corresponding `option()`

With the desired behavior, `USE_VCPKG` needs to be a tristate: explicitly true (so we can warn if vcpkg is not found), explicitly false (so we can ignore vcpkg if it is set), or undefined (so we can go by existence of vcpkg). Thus, I don't declare it as an `option()` - I just document it in the README.

Reply:
If a variable should be set by the user, then it should be an option or cache variable. If it has more than 2 state, then it should be set as string cache variable, ie `USE_VCPKG` as "AUTOMATIC", "FORCE_VCPKG" or "FORCE_SYSTEM".

Reply:
how should that be expressed in the cmake file?

Reply:
Similarly to the example with CRYPTOBACKEND in this doc page https://cmake.org/cmake/help/book/mastering-cmake/chapter/CMake%20Cache.html

Reply:
> Similarly to the example with CRYPTOBACKEND in this doc page https://cmake.org/cmake/help/book/mastering-cmake/chapter/CMake%20Cache.html

Feedback incorporated in https://github.com/DescentDevelopers/Descent3/pull/469/commits/83b21c4a8021a9e16a017cbdc55fb80abafee31b